specification/2.0/README.md

```diff
@@ -2470,14 +2470,14 @@ A set of parameter values that are used to define the metallic-roughness materia
    
 ### material.normalTexture
 
-A tangent space normal map. Each texel represents the XYZ components of a normal vector in tangent space. <TODO: Add more explaination on how XYZ is calculated>.
+A tangent space normal map. The texture contains RGB components in linear space. Each texel represents the XYZ components of a normal vector in tangent space. Red [0 to 255] maps to X [-1 to 1]. Green [0 to 255] maps to Y [-1 to 1]. Blue [128 to 255] maps to Z [0 to 1]. The normal vectors use OpenGL conventions where +X is right and +Y is up. +Z points toward the viewer.
```

Comment:
> [0 to 255] maps to X [-1 to 1]...

I assume that we expecting something like this in GLSL:
```glsl
vec3 norm = tex_norm.rgb * 2.0 - 1.0;
```

So X, Y = 0 aren't representable, and Blue [128, 255] maps to Z [1/255 to 1] right?
 - `127 -> -1/255`
 - `128 -> +1/255`